src/components/pages/landing-de.tsx

```diff
@@ -1,203 +1,250 @@
+      <section>
         <LandingSubjects data={subjectsData} />
-      </SubjectsSection>
+      </section>
 
-      <AboutSection>
+      <section className="mt-12 flex flex-col sm:flex-row">
         <LandingAbout />
-      </AboutSection>
-
-      <Section>
-        <StyledH2>Serlo.org ist die Wikipedia fürs Lernen</StyledH2>
-        <IconStyle>
-          <ParticipateSVG />
-        </IconStyle>
-        <FlexCol>
-          <p className="serlo-p ml-0">
-            Wir suchen Lehrkräfte mit Begeisterung für ihr Fach. Werden Sie
-            Autor*in auf serlo.org, erstellen Sie <b>neue Inhalte</b> und helfen
-            Sie uns, die <b>Qualität</b> der Lernplattform zu sichern.
-          </p>
-          <Button href="/community">
-            Zur Startseite für Autor*innen{' '}
-            <FontAwesomeIcon icon={faArrowCircleRight} size="1x" />
-          </Button>
-        </FlexCol>
-        <FlexCol>
-          <p className="serlo-p ml-0">
-            Wir suchen neue hauptamtliche und ehrenamtliche Teammitglieder für
-            die Bereiche <b>Softwareentwicklung</b>, <b>Redaktion</b> und{' '}
-            <b>NGO-Management</b>.
-          </p>
-          <Button href="/jobs">
-            Jobs und Engagement{' '}
-            <FontAwesomeIcon icon={faArrowCircleRight} size="1x" />
-          </Button>
-        </FlexCol>
-      </Section>
-
-      <PrinciplesSection>
-        <PrinciplesGraphic strings={landingStrings} />
-      </PrinciplesSection>
-
-      <Section>
-        <StyledH2>Werden Sie Teil unserer Bewegung für freie Bildung</StyledH2>
-        <IconStyle>
-          <DonateSVG />
-        </IconStyle>
-        <FlexCol>
-          <p className="serlo-p ml-0">
-            Bildung gehört uns allen! Werden Sie Mitglied in unserer
-            Organisation Serlo Education e.V. und so zu Mitherausgeber*in der
-            freien Lernplattform.
-          </p>
-          <Button href="/beitreten">
-            Mitglied werden{' '}
-            <FontAwesomeIcon icon={faArrowCircleRight} size="1x" />
-          </Button>
-        </FlexCol>
-        <FlexCol>
-          <p className="serlo-p ml-0">
-            Softwareentwicklung und Lerninhalte erstellen kostet Geld. Wir
-            freuen uns sehr, wenn Sie Serlo mit einer Spende unterstützen.
-          </p>
-          <Button href="/spenden">
-            Spenden <FontAwesomeIcon icon={faArrowCircleRight} size="1x" />
-          </Button>
-        </FlexCol>
-      </Section>
-    </>
+      </section>
+
+      <section>
+        <h3
+          className={clsx(
+            'text-center text-4xl text-truegray-700 font-bold',
+            'leading-cozy tracking-tight',
+            'max-w-2xl mt-40 mx-auto relative z-10'
+          )}
+        >
+          Wir sind eine große, ehrenamtliche Community und gestalten Serlo
+          <p className="text-brand italic font-handwritten">gemeinsam.</p>
+        </h3>
+        <div className="relative z-0 h-0 w-full mt-1">
```

Comment:
Tbh: not a fan of html markup for a decorative element. For me that's hacky.

Reply:
I can understand your idea behind this statement, but the "evidence" points in another direction:

Counter example: https://big.tc/ is using simple plain html elements for the arrows (and is not even setting them as background - just straight img tags).

According to https://www.w3.org/WAI/tutorials/images/decorative/ I'm already using css background for this.

A google search wasn't that supportive of that claim: https://www.google.de/search?q=don't+use+html+for+decorative+elements

So I think, by using css background we are already a bit better than the "default" and it feels ok to me to build it that way. At least it's simple to understand whats happening.